File: 6_multilevel_inheritance.py
class Food:
    def start(self, name):
        print("food Name: ", name)

class Rice(Food):
    def __init__(self, name):
        self.name = name
        super().__init__()

class Mango(Rice):
    def __init__(self, name):
        super().__init__(name)

        print("habi Tasty mango name is ", name)

langora = Mango("langora")
langora.start("rupali")

File: 6_multilevel_inheritance.cpp
#include<bits/stdc++.h>
using namespace std;
class Food {
public:
    void start(string name) {
        cout << "Food Name: " << name << endl;
    }
};

class Rice : public Food {
public:
    string name;
    Rice(string name) : Food() {
        this->name = name;
        start(name);
    }
};

class Mango : public Rice {
public:
    Mango(string name) : Rice(name) {
        cout << "habi tasty mango name is " << name << endl;
    }
};

int main() {
    Mango langra("langora");
    return 0;
}